bucket/altdrag.json

```diff
@@ -0,0 +1,20 @@
+{
+    "homepage": "https://github.com/stefansundin/altdrag",
+    "description": "Easily drag windows when pressing the Alt key",
+    "license": "GPL-3.0-or-later",
+    "version": "1.1",
+    "url": "https://github.com/stefansundin/altdrag/releases/download/v1.1/AltDrag-1.1.zip",
+    "hash": "5e1cf4fd8bfbdeca672cd53141019471b344317c81fdefe1ae9cb3f96183bdf9",
+    "extract_dir": "AltDrag",
```

Comment:
```suggestion
    "extract_dir": "altdrag-1.1",
```

Folder contains version number. Update autoupdate accordingly

![zip](https://i.imgur.com/DnPVkKu.png)

Reply:
There is no exe inside zip tho.

![Zip](https://i.imgur.com/bFNFfNa.png)

Reply:
Sorry, my bad